### PR TITLE
Bug643-Fix adding missing SeeItSayItLabel, reference corresponding GameObject, and regression-prevention UnityTests

### PR DESCRIPTION
**Pull request by ms-RistoRK**
Associated issue: https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/issues/643

### CODE REVIEW

File: org.mixedrealitytoolkit.uxcomponents/CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
-## [3.2.0-development] - 2024-02-04
```
Comment (AMollis) [inline org.mixedrealitytoolkit.uxcomponents/CHANGELOG.md:5]: No need to update the minor version.  The minor version has already been updated.

Before updating versions, check what version was last released: 

https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/releases

File: org.mixedrealitytoolkit.uxcomponents/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "org.mixedrealitytoolkit.uxcomponents",
-  "version": "3.2.0-development",
```
Comment (AMollis) [inline org.mixedrealitytoolkit.uxcomponents/package.json:3]: No need to update the minor version. The minor version has already been updated.

Before updating versions, check what version was last released:

https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/releases

File: org.mixedrealitytoolkit.uxcomponents/CHANGELOG.md (hunk 1 above)
Comment (AMollis) [reply org.mixedrealitytoolkit.uxcomponents/CHANGELOG.md:5]: Note, in the future, if you do need to add information to the changelog for a new version, add a NEW section. Do not edit existing section titles.  The changelog must contain changes for ALL versions.

Note, the changelog files are new...hence why they are empty :)

**Review by AMollis — CHANGES_REQUESTED** [review]

File: org.mixedrealitytoolkit.uxcomponents/CHANGELOG.md (hunk 1 above)
Comment (ms-RistoRK) [reply org.mixedrealitytoolkit.uxcomponents/CHANGELOG.md]: Updated in latest commit.

File: org.mixedrealitytoolkit.uxcomponents/package.json (hunk 2 above)
Comment (ms-RistoRK) [reply org.mixedrealitytoolkit.uxcomponents/package.json]: Updated in latest commit.

**Review by AMollis — APPROVED** [review]

**Pull request merged**